### PR TITLE
fix hover effect

### LINKED ISSUE #311: fix the visibility of slack and github logos
**Current Behavior**

image
image

**Desired Behavior**
    Background will not be changed , only logos will be visiable after hover effect .
image
image

 fix the visibility of slack and github logos on hover in the footer of https://getnighthawk.dev/      

**Resources**

**Alternatives / Additional Context**

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by Rajivkumawat3**
**Description**

This PR fixes #311

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: docs/_includes/social-icon.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
...
...height="24px"><path d="M9,7C8.359,7,4.639,7,4,7C2.895,7,2,7.895,2,9c0,1.105,0.895,2,2,2c0.639,0,4.359,0,5,0c1.105,0,2-0.895,2-2	C11,7.895,10.105,7,9,7z" fill="grey" class="first"/><path d="M11,4c0,0.598,0,2,0,2S9.507,6,9,6C7.895,6,7,5.105,7,4s0.895-2,2-2S11,2.895,11,4z" fill="grey" class="second"/><path d="M7,14c0,0.641,0,4.361,0,5c0,1.105,0.895,2,2,2c1.105,0,2-0.895,2-2c0-0.639,0-4.359,0-5c0-1.105-0.895-2-2-2	C7.895,12,7,12.895,7,14z" fill="grey" class="third"/><path d="M4,12c0.598,0,2,0,2,0s0,1.493,0,2c0,1.105-0.895,2-2,2s-2-0.895-2-2S2.895,12,4,12z" fill="grey" class="fourth"/><path d="M14,16c0.641,0,4.361,0,5,0c1.105,0,2-0.895,2-2c0-1.105-0.895-2-2-2c-0.639,0-4.359,0-5,0c-1.105,0-2,0.895-2,2	C12,15.104,12.895,16,14,16z" fill="grey" class="fifth"/><path d="M12,19c0-0.598,0-2,0-2s1.493,0,2,0c1.105,0,2,0.895,2,2c0,1.105-0.895,2-2,2S12,20.104,12,19z" fill="grey" class="sixth"/><path d="M16,9c0-0.641,0-4.361,0-5c0-1.105-0.895-2-2-2c-1.105,0-2,0.895-2,2c0,0.639,0,4.359,0,5c0,1.105,0.895,2,2,2	C15.104,11,16,10.104,16,9z" fill="grey" class="seventh"/><path d="M19,11c-0.598,0-2,0-2,0s0-1.493,0-2c0-1.105,0.895-2,2-2c1.105,0,2,0.895,2,2S20.104,11,19,11z" fill="grey" class="eighth"/></svg>
```
Comment (vishalvivekm) [inline docs/_includes/social-icon.html:25]: Why is there a need of new svg ? @Rajivkumawat3

**Comment by bharathkalyans** [comment]
The hover effect is not shown on the slack icon @Rajivkumawat3 !

**Comment by vishalvivekm** [comment]
@Rajivkumawat3 this is the preview link: https://deploy-preview-316--getnighthawk.netlify.app/

**Comment by vishalvivekm** [comment]
@Rajivkumawat3 Please take the reference from https://github.com/meshery/meshery.io

**Review by vishalvivekm — CHANGES_REQUESTED** [review]

File: docs/_includes/social-icon.html (hunk 1 above)
Comment (Rajivkumawat3) [reply docs/_includes/social-icon.html:25]: sir it is not new svg , i uppdate current svg ,gave class to each path to fill color of slag-icon

**Comment by vishalvivekm** [comment]
@Rajivkumawat3 
- no color filling in slack icon on hover ?
- github icon again the same issue, on hover the visibility can be improved.

https://github.com/layer5io/getnighthawk/assets/110674407/0f3b5253-4fac-40cf-a902-28f07501b8aa

**Comment by Rajivkumawat3** [comment]
sir link  the preview link: https://deploy-preview-316--getnighthawk.netlify.app/ you provided did not showing the change i made like meshrey.io

**Comment by Rajivkumawat3** [comment]
yes sir on my local host it is showing changes on hover , but when i give a pull request then it is not showing changes here

**Comment by Rajivkumawat3** [comment]
sir check now working fine

**Review by vishalvivekm — APPROVED** [review]
LGTM
Thank you Sir @Rajivkumawat3 🙂

**Comment by vishalvivekm** [comment]
Let's discuss this on the Websites call. Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#) @Rajivkumawat3

**Comment by Rajivkumawat3** [comment]
ok sir

**Review by Chadha93 — CHANGES_REQUESTED** [review]
@Rajivkumawat3 In the deploy preview, the Twitter icon is disabled, and icons are of different sizes. These items require an update.

**Comment by Rajivkumawat3** [comment]
@Chadha93 sir twitter icon is not disable

**Review by Chadha93 — APPROVED** [review]
Thank you @Rajivkumawat3

**Pull request merged**